pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -378,6 +379,8 @@ func readFile(name string, reader io.Reader, templateData interface{}) (finalNam
 		if err != nil {
 			return name, data, err
 		}
+		logrus.Tracef("SHDEBUG Rendering file %s", name)
+		logrus.Tracef("SHDEBUG templateData %s", templateData)
```

Comment:
This is debugging where I'm trying to work out if the release url is available for templating, doesn't seem to be but perhaps we can add it so the script can be templated with the correct RHCOS_IMAGE_URL